src/yetibot/core/handler.clj

```diff
@@ -14,6 +16,36 @@
                                       format-data-structure
                                       format-exception-log]]))
 
+(defn embedded-enabled?
+  "Determine whether or not embedded commands are enabled.
+
+   Embedded commands look like:
+
+   The weather looks nice today `weather seattle`
+
+   instead of the more explicit command execution syntax:
+
+   !weather seattle
+
+   This means multiple embedded commands can be contained within a single
+   message, such as:
+
+   `weather seattle` `weather new york`
+
+   However, some teams may find the overloading of backtick syntax disruptive,
+   so it can be globally disabled in configuration.
+
```

Comment:
Excellent.  I think it's a great feature, but we make such heavy use of fixed-width formatting that it triggers too many random operations for our main channels.